runtime/runtimeUtil.py

```diff
@@ -14,7 +14,7 @@ class RUNTIME_CONFIG(Enum):
     TEST_OUTPUT_DIR             = "test_outputs/"
     VERSION_MAJOR               = 1
```

Comment:
Lets just change major to 2, in preparation for full release next season.

Reply:
Should keep for internal use to distinguish between runtime changes to make sure newest code is utilized